Kitodo/src/main/java/org/kitodo/forms/CurrentTaskForm.java

```diff
@@ -429,6 +432,46 @@ private void download() {
         }
     }
 
+    /**
+     * Regenerate all images.
+     */
+    public void regenerateAllImagesButtonClick() {
```

Comment:
Why the ButtonClick appendix?

Reply:
Because it is the method to deal with the event that `regenerateAllImagesButton` is clicked. The method responds to the click event, by adding a long running task to the task manager, which then generates the images. The method does not generate the images itself, so naming it `regenerateAllImages()` would be wrong.

Reply:
We never add ButtonClick to method names and in the form classes most of them are activated by button click....

Reply:
We are still missing proper form classes. There is a ticket for this and an intranet page where I documented a suggestion for standardization: https://intranet.slub-dresden.de/display/KITPROD/JSF+Formularklassen In short, the method name is the component name + the event it responds to. Here `Button` is part of the component name `regenerateAllImagesButton` and `Click` is the event. If there was a method that deals with drag and drop onto the button (end drag event), it would be called `regenerateAllImagesButtonEndDrag(...)`